Steer/DigitizerWorkflow/src/SimpleDigitizerWorkflow.cxx

```diff
@@ -516,6 +516,11 @@ WorkflowSpec defineDataProcessing(ConfigContext const& configcontext)
     auto useCCDB = configcontext.options().get<bool>("use-ccdb-tof");
     auto ccdb_url_tof = o2::base::NameConf::getCCDBServer();
     auto timestamp = configcontext.options().get<int>("timestamp-tof");
+
+    if (timestamp < 0) {
+      timestamp = o2::raw::HBFUtils::Instance().startTime / 1000;
+    }
```

Comment:
For the record: the HBFUtils.startTime at the moment is 0, unless it is set explicitly via command line option. Is it ok for what you want to do with the timestamp?

Reply:
timestamp is a parameter of the workflow (default = -1)
In this way I can overwrite the HBFUtils in a local test in case I need it

Reply:
overwrite=means use my own timestamp instead of the one provided by utils

Reply:
@noferini ok, but the fullCI is crashing after failing to fetch a TOF CCDB object, I guess you need to upload a default object with infinite coverage.

Reply:
That's strange (default entry is supposed to be present). I will have a look

Reply:
Hi @noferini ,
Indeed, the problem appears to be different:
````
89783:ZDCDigitizer]: [18:41:05][WARN] CCDB: Did not find an alien token; Cannot serve objects located on alien:/
````
leads to
````
[ERROR] SEVERE: Device TOFDigitizer (89779) had at least one message above severity fatal: Got nullptr from CCDB for path TOF/Calib/ChannelCalib and timestamp 1641234700000
````
@davidrohr, @TimoWilken: after disabling the http fetching, we need to have an alien token initialized on the build servers...